text.cpp

```diff
@@ -91,7 +91,7 @@ class Text {
       }
 
       auto isMatchingSymbol = [&](const Symbol& symbol) -> bool {
-        return symbol.id == static_cast<uint32_t>(c);
+          return symbol.id == static_cast<unsigned char>(c);
```

Comment:
Yes. Also if the type is unsigned here it should probably also be like that in the zengl struct (which is currently `int32_t`).
However, I don't know how unicode works in C++ and if zengl files even use unicode (which I kind of hope).